### PR TITLE
[V2 Docs] Update usage section

### PR DESCRIPTION
**Pull request by troch**
- [x] Remove Inferno / Preact pages
- [x] Add "Pushing to props"
- [x] Add "Rendering components"

### CODE REVIEW

**Comment by troch** [comment]
@jtmcgrath ready to be reviewed

File: docs/usage/pushing-to-props.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+export default withEffects(handler)(aperture)(DoubleValue)
+```
+
+## Replacing props
+
+`asProps` is used exactly like `toProps`, except that the provided props will be the only ones passed to the child component.
+
+It allows to fully control what props are passed through, and can result in performance benefits by controlling exactly when a component re-renders.
+
+Essentially, `toProps` and `asProps` allow to inject data into components, by using existing props or external data sources (sideways data loading).
```
[the remark this answers is not in the document]
Comment (troch) [reply docs/usage/pushing-to-props.md]: https://stackoverflow.com/questions/31583501/what-is-sideways-data-loading-in-React I'm not the only one hehe

File: docs/usage/pushing-to-props.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+## Stateful apertures
+
+The ability to set component props combined with the ability to listen to events (with `pushEvent` and `component.event(name)`), comes the ability of handling state: events are the source of truth, and state is a projection of these events.
+
+The example below is a simple counter example: each time a button is clicked, the count is incremented. We use a reducer to persist state between events, and pass it as props. It will sound familiar if you've used Redux: we go from events to state to props, the same way Redux (with `connect`) goes from actions to state to props (see [Replacing react-redux connect HoC](../recipes/replacing-connect.md]) recipe).
```
[the remark this answers is not in the document]
Comment (troch) [reply docs/usage/pushing-to-props.md]: Good point

File: docs/usage/pushing-to-props.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jtmcgrath) [reply docs/usage/pushing-to-props.md]: What did you do?! 😱 😱 😱 😱 😱 

image

File: docs/usage/pushing-to-props.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ Refract has three built-in effect handlers, to:
 *   Replace props
 *   Render components
 
-This section focuses on adding and replacing props, and its applications. All React, Preact and Inferno packages export a `toProps` function and a `asProps` one. Both these functions take an object of props.
+This section focuses on adding and replacing props, and its applications. All React, Preact and Inferno packages export two effect creators `toProps` and `asProps` one. They both take an object of props.
```
Comment (jtmcgrath) [inline docs/usage/pushing-to-props.md]:
```
`two effect creators `toProps` and `asProps` one
```
->
```
two effect creators: `toProps` and `asProps`
```

**Pull request merged**